File: kotlin/2469.kt
fun convertTemperature(celsius: Double): DoubleArray {
    val fahrenheit = (celsius * 1.8) + 32.00
    val kelvin = celsius + 273.15
    return doubleArrayOf(kelvin, fahrenheit)
}

fun main() {
    println(convertTemperature(36.50).joinToString())
    println(convertTemperature(122.11).joinToString())
}

File: javascript/2469.js

var convertTemp = function(celsius) {
    const kelvin = (celsius + 273.15).toFixed(5);
    const fahrenheit = ((celsius * 1.8 ) + 32).toFixed(5)
    return [kelvin, fahrenheit]
}

convertTemp(36.50)
convertTemp(122.11)